### ISSUE TITLE
German translation

### ISSUE THREAD

**Issue by buster-one** [shaarli/Shaarli#1111]
So far there is no German translation. I would like to implement that, if interested?

**Comment by virtualtam** [comment]
Hi @buster-one, that would be great!

You can find the translation guide here: https://shaarli.readthedocs.io/en/master/Translations/

**Comment by buster-one** [comment]
I am glad that I can contribute to this project. I will check the guide and then get started.

**Comment by virtualtam** [comment]
Thanks, let us know if you need any help using Poedit or submitting a Pull Request on Github :)

**Comment by buster-one** [comment]
Are the german entries added as new lines, or i just replace the entry french with german?

shaarli.po
```
#: application/Languages.php:162
msgid "English"
msgstr ""

#: application/Languages.php:163
msgid "French"
msgstr ""
```

**Comment by virtualtam** [comment]
A new translation file should be created, using the existing French translation as a template.

There are instructions in the docs to add a new language: https://shaarli.readthedocs.io/en/master/Translations/#add-a-new-language

Note that you should not edit the `.po` files directly, and use Poedit instead for both convenience and spell-checking.

To create the German translation:
- prerequisites:
    - Poedit is installed
- start Poedit
- choose `Create New Translation`
- navigate to the existing French translation and select it to use it as a template:
    - it is located under `<your shaarli directory>/inc/languages/fr/LC_MESSAGES/shaarli.po`
- a dialog box will open, asking what language the new translation is in:
    - here, you should select `German`
    - optionally, install the German dictionary for spell checking
- save the new German translation file under:
    -  `<your shaarli directory>/inc/languages/de/LC_MESSAGES/shaarli.po`

Upon each save, Poedit will compile the translation and generate the corresponding `shaarli.mo` machine object file.

To test the translation, a configured web server is required, as well as a development environment to build Shaarli from source code:
- https://shaarli.readthedocs.io/en/master/Server-requirements/
- https://shaarli.readthedocs.io/en/master/Server-configuration/
- https://shaarli.readthedocs.io/en/master/Download-and-Installation/#development-version-mainline

**Comment by virtualtam** [comment]
Closed by #1114
